### ISSUE TITLE
Digiline touchscreen crash

### ISSUE THREAD

**Issue by thomasrudin** [pandorabox-io/pandorabox.io#343]
```
digiline_send('t', { 'adddropdown', 'd', 'p', 3, 4, 4, 1, 1, { 'barley', 'multigrain', 'oat', 'rye', 'wheat' } })
```

**Comment by OgelGames** [comment]
This works:
```
digiline_send('t', { command = "adddropdown", name = "d", X = 3, Y = 4, W = 4, H = 1, selected_id = 1, choices = { 'barley', 'multigrain', 'oat', 'rye', 'wheat' } })
```
But there should be some kind of check to prevent it from crashing...

**Comment by SwissalpS** [comment]
yes, that's the issue. Input is not sanitised properly.

**Comment by thomasrudin** [comment]
Looks like an easy fix (https://cheapiesystems.com/git/digistuff/tree/touchscreen.lua#n256) but i like to do that upstream and not fork and maintain yet another repo locally... :confounded: 

Cheapie is notified: http://irc.minetest.net/minetest/2019-12-11#i_5621567

**Comment by cheapie** [comment]
I think I have this fixed in f0ce665.

**Comment by thomasrudin** [comment]
Many thanks @cheapie !
But just telling how you accept pull-requests would have sufficed :smile:
